src/Rule.php

```diff
@@ -1115,103 +1028,115 @@ public function showActionsList($rules_id, $options = [])
-            $header_bottom .= "<th width='10'>";
-            $header_bottom .= Html::getCheckAllAsCheckbox('mass' . $this->ruleactionclass . $rand) . "</th>";
         }
 
-        $header_end .= "<th class='center b'>" . _n('Field', 'Fields', Session::getPluralNumber()) . "</th>";
-        $header_end .= "<th class='center b'>" . __('Action type') . "</th>";
-        $header_end .= "<th class='center b'>" . __('Value') . "</th>";
-        $header_end .= "</tr>\n";
-        echo $header_begin . $header_top . $header_end;
+        $massiveactionparams = [
+            'num_displayed'  => min($_SESSION['glpilist_limit'], count($entries)),
+            'check_itemtype' => static::class,
+            'check_items_id' => $rules_id,
+            'container'      => 'mass' . $this->ruleactionclass . $rand,
+            'extraparams'    => [
+                'rule_class_name' => static::class
+            ]
+        ];
 
-        foreach ($this->actions as $action) {
-            $this->showMinimalActionForm($action->fields, $canedit, $rand);
-        }
-        if ($nb) {
-            echo $header_begin . $header_bottom . $header_end;
-        }
-        echo "</table>\n";
+        TemplateRenderer::getInstance()->display('components/datatable.html.twig', [
+            'datatable_id' => 'datatable_ruleaction' . $rules_id . $rand,
+            'is_tab' => true,
+            'nofilter' => true,
+            'columns' => [
+                'field' => _n('Field', 'Fields', Session::getPluralNumber()),
+                'action_type' => __('Action type'),
+                'value' => __('Value')
+            ],
+            'entries' => $entries,
+            'row_class' => 'cursor-pointer',
+            'total_number' => count($entries),
+            'filtered_number' => count($entries),
+            'showmassiveactions' => $canedit,
+            'massiveactionparams' => $massiveactionparams,
+        ]);
 
-        if ($canedit && $nb) {
-            $massiveactionparams['ontop'] = false;
-            Html::showMassiveActions($massiveactionparams);
-            Html::closeForm();
+        if ($canedit) {
+            $rule_class = static::class;
+            echo Html::scriptBlock(<<<JS
+                $(() => {
+                    $('#datatable_ruleaction{$rules_id}{$rand}').on('click', 'tbody tr', (e) => {
+                        //ignore click in first column (the massive action checkbox)
+                        if ($(e.target).closest('td').is('td:first-child')) {
+                            return;
+                        }
+                        const action_id = $(e.currentTarget).data('id');
+                        if (action_id) {
+                            $('#viewaction{$rules_id}{$rand}').load('/ajax/viewsubitem.php',{
+                                type: "{$this->ruleactionclass}",
+                                parenttype: "{$rule_class}",
+                                rules_id: 154,
```

Comment:
ah ah, here is the issue :D

Reply:
I don't even know how that happened... fixed now though